frontend/packages/core/src/Feedback/error/details.tsx

```diff
@@ -35,70 +39,89 @@ const Accordion = styled(MuiAccordion)({
 });
 
 const AccordionSummary = styled(MuiAccordionSummary)<{ $expanded: boolean }>(
-  {
-    background: "linear-gradient(to right, #DB3615 8px, #FDE9E7 0%)",
-    color: "#0D1030",
-    fontSize: "14px",
-    fontWeight: 400,
-    padding: "12px 16px 12px 24px",
-    minHeight: "fit-content",
-    "& .MuiAccordionSummary-content": {
-      margin: "0",
-      alignItems: "center",
-    },
-    "&.MuiAccordionSummary-root.Mui-expanded": {
-      minHeight: "unset",
-    },
+  () => {
+    const theme = useTheme();
+    return {
+      background: `linear-gradient(to right, ${theme.palette.error[600]} 8px, ${theme.palette.error[100]} 0%)`,
+      color: theme.palette.secondary[900],
+      fontSize: "14px",
+      fontWeight: 400,
+      padding: "12px 16px 12px 24px",
+      minHeight: "fit-content",
+      "& .MuiAccordionSummary-content": {
+        margin: "0",
+        alignItems: "center",
+      },
+      "&.MuiAccordionSummary-root.Mui-expanded": {
+        minHeight: "unset",
+      },
+    };
   },
   props => ({
     borderBottomLeftRadius: props.$expanded ? "0" : "8px",
     borderBottomRightRadius: props.$expanded ? "0" : "8px",
   })
 );
 
-const AccordionDetails = styled(MuiAccordionDetails)({
-  background: "linear-gradient(to right, #DB3615 8px, #FFFFFF 0%)",
-  padding: "0",
-  paddingLeft: "8px",
-  borderBottomLeftRadius: "8px",
-  borderBottomRightRadius: "8px",
-  display: "flex",
-  flexDirection: "column",
+const AccordionDetails = styled(MuiAccordionDetails)(() => {
+  const theme = useTheme();
+  return {
+    background: `linear-gradient(to right, ${theme.palette.error[600]} 8px, ${theme.palette.common.white} 0%)`,
+    padding: "0",
+    paddingLeft: "8px",
+    borderBottomLeftRadius: "8px",
+    borderBottomRightRadius: "8px",
+    display: "flex",
+    flexDirection: "column",
+  };
 });
 
-const ListItem = styled("li")({
-  "::marker": {
-    color: "rgba(13, 16, 48, 0.6)",
-  },
-  padding: "2px 0",
+const ListItem = styled("li")(() => {
+  const theme = useTheme();
+  return {
+    "::marker": {
+      color: theme.palette.secondary[700],
```

Comment:
I wonder if we should use the alpha package from material to make it easier to understand and could change in the future.

```
import { alpha } from "@mui/material";

color: alpha(theme.palette.secondary[700], 0.66)
```

What do you think about standardizing to this? I feel like it will be a little easier to understand at just a glance what its doing vs concating the opacity variable.

Reply:
sounds good @jdslaugh , updates have been pushed with alpha